app/pubspec.lock

```diff
@@ -36,6 +50,41 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "2.1.0-nullsafety.1"
+  build:
+    dependency: transitive
+    description:
+      name: build
+      url: "https://pub.dartlang.org"
+    source: hosted
+    version: "1.6.1"
+  built_collection:
+    dependency: transitive
+    description:
+      name: built_collection
+      url: "https://pub.dartlang.org"
+    source: hosted
+    version: "4.3.2"
+  built_value:
+    dependency: transitive
+    description:
+      name: built_value
+      url: "https://pub.dartlang.org"
+    source: hosted
+    version: "7.1.0"
+  camera:
```

Comment:
This aren't mean to be here!!!

- [ ] Delete `pubspec.lock` and run `flutter pub get/ upgrade` to add it back

Reply:
weird that it's here. Noted and will remove it

Reply:
I run the given commands the thing still persist